airbyte-integrations/connectors/source-typeform/source_typeform/source.py

```diff
@@ -204,20 +204,20 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
             form_ids = config.get("form_ids", []).copy()
             # verify if form inputted by user is valid
             try:
-                url = f"{TypeformStream.url_base}/me"
+                url = f"{TypeformStream.url_base}me"
                 auth_headers = {"Authorization": f"Bearer {config['token']}"}
                 session = requests.get(url, headers=auth_headers)
                 session.raise_for_status()
-            except requests.exceptions.BaseHTTPError as e:
+            except Exception as e:
```

Comment:
Why are you changing the exception type error to the generic one?

Reply:
in this case resolving **_config['token']_** if **_token_** is not present was not being handled and the exception was being propagated further down the call stack. it is also the approach that connectors such as Stripe, Mixpanel, etc are using.